.packit.yml

```
@@ -0,0 +1,36 @@
+upstream_package_name: rhc
+downstream_package_name: rhc
+specfile_path: rhc.spec
+
+srpm_build_deps:
+  - gawk
+  - git-core
+  - golang
+  - make
+  - 'pkgconfig(dbus-1)'
+  - 'pkgconfig(systemd)'
+  - rpm-build
+
+actions:
+  post-upstream-clone:
+    - mkdir out
+    - bash -c 'make -f .copr/Makefile spec'
+    - bash -c 'make -f .copr/Makefile tarball'
```

minor nit: you can call two makefile targets at the same time:
```suggestion
    - bash -c 'make -f .copr/Makefile spec tarball'
```